### DISCUSSION TITLE
createSignedUrls() not working. function doesn't exist

### DISCUSSION THREAD

**Discussion by EdgarsLv** [supabase/supabase#5849] [Questions] [answered]
With policies everything is ok.
I can create url with createSignedUrl() copying one path from given array so paths are correct.

![image](https://user-images.githubusercontent.com/56837461/157134535-7468e332-e297-47cf-8d5e-aa883c2dbdaa.png)

for single url pops up info

![image](https://user-images.githubusercontent.com/56837461/157135532-4b8ad85c-c1d0-4adb-ae83-a6081fac4c22.png)

for urls doesn't not.

**Comment by GaryAustin1** [comment, marked as answer]
createSignedUrls is an addition as of a couple of weeks ago, I believe.  Make sure you are running the latest supabase.js.  It works for me on that.

**Reply by EdgarsLv** [reply]
Yes that was the case. Updated and now works.
